### PR TITLE
Update testing setup to use jest

### PR DESCRIPTION
**Pull request by lifeart**
part 1 of https://github.com/stefanpenner/broccoli-concat-analyser/pull/52/files

### CODE REVIEW

File: lib/summarize.js [hunk 1]
```diff
@@ -67,7 +72,7 @@ module.exports = function summarize(summaryPath) {
       if (isUglifyable) {
         return uglify(concatenatedContent);
       } else {
-        return concatenatedContent;
+        return normalizeLineEndings(concatenatedContent);
```
Comment (simonihmig) [inline lib/summarize.js:75]: Could we apply `normalizeLineEndings()` at line 43, in which case I believe we wouldn't have to do it here and at L87 anymore, right?

**Review by simonihmig — COMMENTED** [review]
LGTM, just a minor remark!

**Review by simonihmig — APPROVED** [review]
LGTM, thank you @lifeart! (didn't notice you already updated the PR according to my feedback, sorry for the delay!)

@stefanpenner do yo want to have a second look? Otherwise I would gladly merge this.

**Comment by lifeart** [comment]
UP
cc @stefanpenner

**Comment by simonihmig** [comment]
@lifeart merged this! Can you please rebase your next PR, so we get a clean diff / commit history? Thanks again for your work! 👍

**Pull request merged**